contributors/guide/contributor-cheatsheet/README-hi.md

```diff
@@ -0,0 +1,322 @@
+# कुबेरनेट्स योगदानकर्ता चीट शीट
+
+कुबेरनेट्स परियोजना में उपयोग किए जाने वाले कुबेरनेट्स,टिप्स, ट्रिक्स और सामान्य सर्वोत्तम प्रथाओं में योगदान करते समय सामान्य संसाधनों की एक सूची। यह एक "TL;DR" या आपके GitHub योगदान अनुभव को बेहतर बनाने के लिए उपयोगी जानकारी का त्वरित संदर्भ है।
```

Comment:
1. `कुबेरनेट्स` => `कुबरनेट्स`
2. I don't remember correctly but I think at some places we have used `प्रॉजेक्ट` instead of `परियोजना`
3. Spacing between `कुबेरनेट्स,टिप्स`
4. Any easy words for `सामान्य सर्वोत्तम प्रथाओं`

We can remove this `एक "TL;DR" या`, the sentence will be short and simple.

Reply:
Done.!